pages/common/ocrmypdf.md

```diff
@@ -0,0 +1,28 @@
+# ocrmypdf
+
+> Generate a searchable PDF or PDF/A from a scanned PDF or an image of text.
+> More information: <https://ocrmypdf.readthedocs.io/en/latest/introduction.html>.
+
+- Create a new searchable PDF/A file from a scanned PDF or image file:
+
+`ocrmypdf {{input_file}} {{output.pdf}}`
+
+- Overwrite a scanned PDF file with a searchable PDF file, if successful:
```

Comment:
```suggestion
- Overwrite a scanned PDF file with a searchable PDF file:
```

Could also just reference it just "scanning a file", ignoring the fact it overwrites.

Reply:
Will use "Replace", not "Overwrite" and remove ", if successful".

Reply:
Replace is better, but still don't need the if successful